lib/pure/times.nim

```diff
@@ -1333,14 +1333,13 @@ proc now*(): DateTime {.tags: [TimeEffect], benign.} =
   ##    `cpuTime` instead, depending on the use case.
   getTime().local
 
-proc initDateTime*(monthday: MonthdayRange, month: Month, year: int,
-                   hour: HourRange, minute: MinuteRange, second: SecondRange,
-                   nanosecond: NanosecondRange,
-                   zone: Timezone = local()): DateTime =
+proc dateTime*(year: int, month: Month, monthday: MonthdayRange, 
```

Comment:
This naming goes against Nim's conventions :/

Reply:
it's better than initDateTime2.

Reply:
I guess, but maybe we should document this as being "allowed but discouraged" in our style guide

Reply:
maybe; this can be addressed and discussed in a separate PR though